SYCL/Config/select_device_config.txt

```diff
@@ -0,0 +1 @@
+DeviceName:{{Intel\(R\) Iris\(R\) Xe Graphics \[0x9a49\]}},DriverVersion:{{22.06.22433}}
```

Comment:
Please remove this file and binary - [SYCL/Config/select_device.out](https://github.com/intel/llvm-test-suite/pull/1086/files#diff-da47d5146061561b9b247b172f776a83e0fa6034ae417fe3194fb5f20fc7f49b)